frontend/app/controllers/Joiner.scala

```diff
@@ -204,6 +204,8 @@ trait Joiner extends Controller with ActivityTracking with LazyLogging {
       case _ => Future.successful(None)
     }
 
+    val cookiesToDiscard = List(DiscardingCookie("GU_MEM"), DiscardingCookie("gu_paying_member", "/", Some(Config.guardianShortDomain)))
```

Comment:
I'd consider defining this in Config.scala or somewhere to avoid having to repeat it.